### PR TITLE
Switch from twisted logging back to python logging module

### PR DESCRIPTION
**Pull request by tynanford**
This addresses https://github.com/ChannelFinder/recsync/issues/96

I've tried to get twisted logging to use the conf file loglevel but it hasn't been successful. These attempts mostly revolved around adding this logic to application.py

```python
...

class LogLevelFilter(object):
    def __init__(self, level):
        self.level = level

    def __call__(self, event):
        return event if event.get('log_level') >= self.level else None

...

@implementer(service.IServiceMaker, plugin.IPlugin)
class Maker(object):

...

    lvlname = conf.get('loglevel', 'WARN')
    if lvlname == 'DEBUG':
        lvl = LogLevel.debug
    elif lvlname == 'INFO':
        lvl = LogLevel.info
    elif lvlname == 'WARN':
        lvl = LogLevel.warn
    elif lvlname == 'ERROR':
        lvl = LogLevel.error
    elif lvlname == 'CRITICAL':
        lvl = LogLevel.critical
    else:
        print("Invalid loglevel {}. Will default to level WARN".format(lvlname))
        lvl = LogLevel.warn

    globalLogPublisher.addObserver(LogLevelFilter(lvl))
...
```

It works fine for some log messages but certain log messages like ` _log.debug('announce to {s}'.format(s=A))`  show up in the logs no matter what. So I suggest we revert back to the python logging module unless someone else can figure out the twisted logging since IMO this issue is a blocker to creating a new release.

### CODE REVIEW

File: server/recceiver/cfstore.py [hunk 1]
```diff
@@ -228,19 +228,19 @@ def _commitWithThread(self, TR):
...
 
         pvInfoByName = {}
         for rid, (info) in pvInfo.items():
             if info["pvName"] in pvInfoByName:
-                _log.warn("Commit contains multiple records with PV name: {pv} ({iocid})", pv=info["pvName"], iocid=iocid)
+                _log.warning("Commit contains multiple records with PV name: {pv} ({iocid})".format(pv=info["pvName"], iocid=iocid))
                 continue
             pvInfoByName[info["pvName"]] = info
-            _log.debug("Add record: {rid}: {info}", rid=rid, info=info)
+            _log.debug("Add record: {rid}: {info}".warning(rid=rid, info=info))
```
Comment (jacomago) [inline server/recceiver/cfstore.py]: should be .format not .warning

**Comment by jacomago** [comment]
I don't see any info logs with this change :/

Before:

```
2024-11-14T13:55:06+0000 [recceiver.processors#debug] Available plugin: show
2024-11-14T13:55:06+0000 [recceiver.processors#debug] Available plugin: db
2024-11-14T13:55:06+0000 [recceiver.processors#debug] Available plugin: cf
2024-11-14T13:55:06+0000 [recceiver.cfstore#info] CF_INIT cf
�2024-11-14T13:55:06+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 24.10.0 (/home/recceiver/venv/bin/python 3.11.10) starting up.
�2024-11-14T13:55:06+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.pollreactor.PollReactor.
2024-11-14T13:55:06+0000 [recceiver.application#info] Starting RecService
2024-11-14T13:55:06+0000 [-] CastFactory starting on 40447
�2024-11-14T13:55:06+0000 [recceiver.recast.CastFactory#info] Starting factory <recceiver.recast.CastFactory object at 0xffff8fc66690>
2024-11-14T13:55:06+0000 [-] CastFactory starting on 34243
�2024-11-14T13:55:06+0000 [recceiver.application#info] RecService listening on IPv4Address(type='TCP', host='0.0.0.0', port=34243)
2024-11-14T13:55:06+0000 [-] Announcer starting on 53258
2024-11-14T13:55:06+0000 [-] Starting protocol <recceiver.announce.Announcer object at 0xffff8fe6df90>
2024-11-14T13:55:06+0000 [recceiver.announce#info] Setup Announcer
2024-11-14T13:55:06+0000 [recceiver.cfstore#info] CF_START
2024-11-14T13:55:06+0000 [recceiver.cfstore#debug] Unable to format event {'log_logger': <Logger 'recceiver.cfstore'>, 'log_level': <LogLevel=debug>, 'log_namespace': 'recceiver.cfstore', 'log_source': None, 'log_format': "WHITELIST = {'archive', 'foo', 'bar', 'blah'}", 'log_time': 1731592506.797391}: "'archive', 'foo', 'bar', 'blah'"
2024-11-14T13:55:06+0000 [recceiver.cfstore#info] CF Clean Started
2024-11-14T13:55:06+0000 [recceiver.cfstore#info] CF Clean Completed
```

After:

```
recc1 standard logging

�2024-11-14T13:47:54+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 24.10.0 (/home/recceiver/venv/bin/python 3.11.10) starting up.
�2024-11-14T13:47:54+0000 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.pollreactor.PollReactor.
2024-11-14T13:47:54+0000 [-] CastFactory starting on 42431
�2024-11-14T13:47:54+0000 [recceiver.recast.CastFactory#info] Starting factory <recceiver.recast.CastFactory object at 0xffffb2a08750>
2024-11-14T13:47:54+0000 [-] CastFactory starting on 35687
2024-11-14T13:47:54+0000 [-] Announcer starting on 55958
2024-11-14T13:47:54+0000 [-] Starting protocol <recceiver.announce.Announcer object at 0xffffb2a08a90>
```

**Comment by tynanford** [comment]
Interesting. loglevel=INFO in your "after" logs?

This is when I set loglevel=WARN
```
2024-11-14T09:07:12-0800 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 23.10.0 (/usr/bin/python3 3.12.7) starting up.
2024-11-14T09:07:12-0800 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.pollreactor.PollReactor.
2024-11-14T09:07:12-0800 [-] CastFactory starting on 45722
2024-11-14T09:07:12-0800 [recceiver.recast.CastFactory#info] Starting factory <recceiver.recast.CastFactory object at 0x7f46c1da1640>
2024-11-14T09:07:12-0800 [-] Announcer starting on 45722
2024-11-14T09:07:12-0800 [-] Starting protocol <recceiver.announce.Announcer object at 0x7f46c19f35c0>
```

This is when I set loglevel=INFO
```
2024-11-14T09:07:41-0800 [twisted.scripts._twistd_unix.UnixAppLogger#info] twistd 23.10.0 (/usr/bin/python3 3.12.7) starting up.
2024-11-14T09:07:41-0800 [twisted.scripts._twistd_unix.UnixAppLogger#info] reactor class: twisted.internet.pollreactor.PollReactor.
2024-11-14T09:07:41-0800 [-] INFO:recceiver.application Starting RecService
2024-11-14T09:07:41-0800 [-] CastFactory starting on 45722
2024-11-14T09:07:41-0800 [recceiver.recast.CastFactory#info] Starting factory <recceiver.recast.CastFactory object at 0x7fb294242b70>
2024-11-14T09:07:41-0800 [-] INFO:recceiver.application RecService listening on IPv4Address(type='TCP', host='0.0.0.0', port=45722)
2024-11-14T09:07:41-0800 [-] Announcer starting on 45722
2024-11-14T09:07:41-0800 [-] Starting protocol <recceiver.announce.Announcer object at 0x7fb2942bff50>
2024-11-14T09:07:41-0800 [-] INFO:recceiver.announce Setup Announcer
2024-11-14T09:07:41-0800 [-] INFO:recceiver.cfstore CF_START
2024-11-14T09:07:41-0800 [-] INFO:recceiver.cfstore CF Clean Started
2024-11-14T09:07:41-0800 [-] INFO:recceiver.cfstore CF Clean Completed
```

**Comment by jacomago** [comment]
Got it! WARN is default.

**Review by jacomago — CHANGES_REQUESTED** [review]

File: server/recceiver/cfstore.py (hunk 1 above)
Comment (tynanford) [reply server/recceiver/cfstore.py]: Ah yes, great catch! Fixed

**Review by jacomago — APPROVED** [review]

**Pull request merged**